package-lock.json

```diff
@@ -0,0 +1,1875 @@
+{
```

Comment:
Hm. I'm a bit unfamiliar with how this file works. @stephenplusplus is this file correct?